extensions/amp-analytics/0.1/test/test-iframe-transport.js

```diff
@@ -115,5 +116,17 @@ describes.realWin('amp-analytics.iframe-transport', {amp: true}, env => {
     expect(frame2.usageCount).to.equal(0);
     expect(env.win.document.getElementsByTagName('IFRAME')).to.have.lengthOf(0);
   });
-});
 
+  it('gets correct client lib URL in local/test mode', () => {
+    const url = iframeTransport.getLibScriptUrl();
+    expect(url).to.contain(env.win.location.host);
+    expect(url).to.contain('/dist/iframe-transport-client-lib.js');
+  });
+
+  it('gets correct client lib URL in prod mode', () => {
+    const url = iframeTransport.getLibScriptUrl(true);
+    expect(url).to.contain(urls.thirdParty);
+    expect(url).to.contain('/iframe-transport-client-v0.js');
+    expect(url).to.match(/https:\/\/3p\.ampproject\.net\/\$internalRuntimeVersion\$\/iframe-transport-client-v0\.js/); // eslint-disable-line max-len
```

Comment:
Why not just do a string compare which will be more readable and can be split across lines allowing for removal of eslint override?

expect(url).to.equal('https://3p.ampproject.net/$internalRuntimeVersion$/iframe-transport-client-v0.js');

Reply:
Good call. I originally wrote \d+ instead of $internalRuntimeVersion$, thinking that substitution would have already happened. Should have changed to .equal() when switching away from \d+.